extras/bindata.bzl

```diff
@@ -65,6 +68,7 @@ bindata = go_rule(
             cfg = "data",
         ),
         "package": attr.string(mandatory = True),
+        "experimental_strip_extenal": attr.bool(default = False),
```

Comment:
This exposes too much implementation detail in the API. Maybe a more general attribute, `prefix` would be better?

Reply:
While it would be possible to add a `prefix` attribute, this solves a more general problem not limited to `rules_go` and the scope of this PR. I'm not sure if `bindata` works without `prefix` for external repositories and I don't think `prefix` would be used by a lot of users, leaving consumers of these rules with broken artifacts.

I hoped we could turn this on by default eventually and remove it afterward. WDYT?